src/hotspot/share/runtime/globals.hpp

```diff
@@ -1071,6 +1071,10 @@ const intx ObjectAlignmentInBytes = 8;
   product(bool, UseHeavyMonitors, false,                                    \
           "use heavyweight instead of lightweight Java monitors")           \
```

Comment:
For deprecated flags, make the description:
"(Deprecated) Use heavyweight instead of lightweight Java monitors"

Reply:
Also there's some test somewhere that you have to add this flag to.  @dholmes-ora would know where that is.

Reply:
`./runtime/CommandLine/VMDeprecatedOptions.java` :)